app/config/development.js

```diff
@@ -37,30 +38,29 @@ module.exports = Object.freeze({
         password: '',
     },
     emailValidator: {
-        callbackEndpoint: 'http://localhost:8080/callback/validationEmail',
         /* send grid */
         apiKey: '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',
         username: 'no-reply@litentry.com',
         subject: 'Validation From Litentry',
-        jobInterval: 30,        //  seconds
+        jobInterval: 30, //  seconds
     },
 
     elementValidator: {
-        callbackEndpoint: 'http://localhost:8080/callback/validationElement',
         roomId: '!fwdbUHBppFPYOJXzNx:matrix.org',
-        accessToken: 'MDAxOGxvY2F0aW9uIG1hdHJpeC5vcmcKMDAxM2lkZW50aWZpZXIga2V5CjAwMTBjaWQgZ2VuID0gMQowMDJiY2lkIHVzZXJfaWQgPSBAbGl0ZW50cnktYm90Om1hdHJpeC5vcmcKMDAxNmNpZCB0eXBlID0gYWNjZXNzCjAwMjFjaWQgbm9uY2UgPSA3fmxqWWorOXc1WnBGb0lNCjAwMmZzaWduYXR1cmUg3oXZA9SOsJE2bXUEnelgKxIbFJrqn5kFzv1LHgk3U4UK',
+        accessToken:
```

Comment:
This is the development file, should be fine no?

Reply:
Theoretically, the development/staging/production config should be different. However, we keep them the same in this project at this moment, since the third-part services in the development config are not in a sandbox environment. Take twitter configuration as an example, even we distinguish dev/staging/production accounts, the development account can still be used to post tweets or send direct messages to twitter users.

Reply:
I see, at least let remove them from this commit, then.

Reply:
I guess they should be changed if possible then because this file was not encrypted (It is already possible for any user to see the content). 

The changes on the accessTokens are just formatting changes (prettier).

Reply:
Definitely, we need to create new `accessToken`, `apiKey` etc.

Reply:
I created a new access token, apikey, etc. and encrypted them.